wp-content/themes/core/components/blocks/hero/Controller.php

```diff
@@ -0,0 +1,167 @@
+<?php
```

Comment:
Totally cool with you merging this Mike and then cleaning it up (given where we've landed as of today in our approach) post merge since we still need to do the blocks as the final thing.